wallets/react-web3wallet/README.md

```diff
@@ -0,0 +1,261 @@
+# Web3Wallet Example (React, Typescript, Ethers, NextJS, Cosmos)
+
+This example aims to demonstrate basic and advanced use cases enabled by WalletConnect's Web3Wallet SDK.
+
+**This Web3Wallet example implementation** is to serve as a reference for wallet developers.
+
+Please only use this for reference and development purposes, otherwise you are at risk of losing your funds.
+
+# Useful links
+
+🔗 Live Web3Wallet app - https://react-web3wallet.vercel.app <br />
+🔗 Live dapp - `Sign` - https://react-app.walletconnect.com <br />
+🔗 Live dapp - `Auth` - https://react-auth-dapp.walletconnect.com/ <br />
+📚 WalletConnect docs - https://docs.walletconnect.com/2.0
+
+## Getting started
+
+Example is built atop of [NextJS](https://nextjs.org/) in order to abstract complexity of setting up bundlers, routing etc. So there are few steps you need to follow in order to set everything up
+
+1. Go to [WalletConnect Cloud](https://cloud.walletconnect.com/sign-in) and obtain a project id
+
+2. Add your project details in [WalletConnectUtil.ts](https://github.com/WalletConnect/web-examples/blob/main/wallets/react-wallet-v2/src/utils/WalletConnectUtil.ts) file
+
+3. Install dependencies `yarn install` or `npm install`
+
+4. Setup your environment variables
+
+```bash
+cp .env.local.example .env.local
+```
+
+Your `.env.local` now contains the following environment variables:
+
+- `NEXT_PUBLIC_PROJECT_ID` (placeholder) - You can generate your own ProjectId at https://cloud.walletconnect.com
+- `NEXT_PUBLIC_RELAY_URL` (already set)
+
+5. Run `yarn dev` or `npm run dev` to start local development
+
+## Navigating through example
```

Comment:
Think we can just remove this section. All the references and links are to the wallet-v2 implementation